.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        node: ['15', '14', '12']
+        # this is needed only for some plugins
+        # but I don't see how we can disable it without having to remove it when we don't need it
+        hapi: ['20', '19']
+
+    runs-on: ${{ matrix.os }}
+    name: Test on ${{ matrix.os }} using Node@${{ matrix.node }}
```

Comment:
The resulting string is too long to fit on the left-side overview panel here: https://github.com/hapijs/lab-external-module-test/actions/runs/349918210

At least remove the `Test on`, which is already included in the `name`.

Additionally, the `matrix.os` entry can be shortened, if the `-latest` part is removed and added to the `runs-on` entry instead.

Reply:
Yes indeed. Good suggestion I'll make changes based on what @lloydbenson did on the 2 other PR.